packages/api/es/indexer.js

```diff
@@ -394,12 +413,21 @@ async function granule(esClient, payload, index = defaultIndexAlias, type = 'gra
         files: uniqBy(g.files, 'filename'),
         error: exception,
         createdAt: get(payload, 'cumulus_meta.workflow_start_time'),
-        timestamp: Date.now()
+        timestamp: Date.now(),
+        productVolume: getGranuleProductVolume(g.files),
+        timeToArchive: get(payload, 'meta.sync_granule_duration'),
+        productionDateTime: get(payload, 'meta.post_to_cmr_end')
```

Comment:
This should be granule's productionDateTime (from Matt's comments).  It is in cmr metadata Granule.DataGranule.ProductionDateTime